src/examples/install_aws_cli.yml

```diff
@@ -0,0 +1,17 @@
+version: 2.1
```

Comment:
This looks similar to this example that already exists: https://circleci.com/orbs/registry/orb/circleci/aws-cli#usage-install-aws-cli

Reply:
I have added this file cause i didn't find an separated example file for it 
i have removed it now since i have found it inside the  repo